src/main/java/org/learn/example/connection/Wallet.java

```diff
@@ -0,0 +1,152 @@
+package org.learn.example.connection;
+
+public class Wallet {
+    private int bill1;
+    private int bill5;
+    private int bill10;
+    private int bill20;
+    private int bill50;
+    private int bill100;
+    private static final int[] number = new int[]{1, 5, 10, 20, 50, 100};
+    private final int[] bills = new int[]{bill1, bill5, bill10, bill20, bill50, bill100};
+
+    public Wallet(int initialMoney, String denomination){
+        setHighestFirst(initialMoney);
+    }
+
+    public int[] getBills(){
+        return new int[]{bill1, bill5, bill10, bill20, bill50, bill100};
```

Comment:
this is fine, since the example is using bill1, bill5, bill10 etc for simplicity, but you can also just make it a separate class to hold currency data